src/handleEvent.ts

```diff
@@ -181,18 +279,68 @@ async function handleEvent(indexer: Indexer, event: Event) {
           .collection(`rounds/${event.address}/projects`)
           .updateById(projectId, (application) => ({
             ...application,
-            status: projectApp.status,
+            status: projectApp.status ?? application.status,
             payoutAddress: projectApp.payoutAddress,
           }));
       }
       break;
     }
 
+    case "ApplicationStatusesUpdated": {
+      const bitmap = new StatusesBitmap(BigInt(128), BigInt(2));
```

Comment:
@boudra  the first param (width) should be `256` which is the maximum in solidity. so that internally it does 
`this._itemsPerRow = 256 / 2; // = 128 `

Reply:
ah cool I understood it as number of statuses, it makes sense